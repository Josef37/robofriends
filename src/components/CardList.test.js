import React from "react";
import { shallow } from "enzyme";
import CardList from "./CardList";

describe("CardList component", () => {
  it("renders empty robots correctly", () => {
    const wrapper = shallow(<CardList robots={[]} />);
    expect(wrapper.debug()).toMatchSnapshot();
  });

  it("renders robots correctly", () => {
    const wrapper = shallow(
      <CardList robots={[
        { id: 1, name: "Robot", email: "robot@mail.com" }, 
        { id: 3, name: "Bot Bot", email: "bot@bot.com" }
      ]} />
    );
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
