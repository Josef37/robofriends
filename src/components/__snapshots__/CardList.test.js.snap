// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList component renders empty robots correctly 1`] = `"<div />"`;

exports[`CardList component renders robots correctly 1`] = `
"<div>
  <Card id={1} name=\\"Robot\\" email=\\"robot@mail.com\\" />
  <Card id={3} name=\\"Bot Bot\\" email=\\"bot@bot.com\\" />
</div>"
`;
